low-level/sensors/impl/generic/node_hw.py

```diff
@@ -802,17 +750,15 @@ def _get_sensor_list_by_type(self, sensor_type):
            the first element is the sensor id and
            the second is the number."""
 
-        sensor_list_out, retcode = self._run_ipmitool_subcommand(f"sdr type '{sensor_type}'")
+        sensor_list_out, err, retcode = self._run_ipmitool_subcommand(f"sdr type '{sensor_type}'")
         out = []
 
         if retcode != 0:
-            if isinstance(sensor_list_out, tuple):
-                sensor_list_out = [val for val in sensor_list_out if val]
-            msg = f"ipmitool sdr type command failed: {b''.join(sensor_list_out)}"
+            msg = f"ipmitool sdr type command failed: {err}"
             logger.warning(msg)
             return out
 
-        sensor_list = b''.join(sensor_list_out).decode(self.IPMI_ENCODING).split("\n")
+        sensor_list = sensor_list_out.split("\n")
```

Comment:
so here we are getting already decoded using required encoding string from _run_ipmitool_subcommand(), so not doing again in consumer code